charts/reporting-operator/templates/custom-resources/report-queries/node-cpu.yaml

```diff
@@ -45,7 +45,53 @@ spec:
 apiVersion: metering.openshift.io/v1alpha1
 kind: ReportGenerationQuery
 metadata:
-  name: "node-cpu-allocatable"
+  name: "node-cpu-capacity"
+  labels:
+    operator-metering: "true"
+{{- block "extraMetadata" . }}
+{{- end }}
+spec:
+  reportQueries:
+  - "node-cpu-capacity-raw"
+  view:
+    disabled: true
+  columns:
+  - name: period_start
+    type: timestamp
+    unit: date
+  - name: period_end
+    type: timestamp
+    unit: date
+  - name: node
+    type: string
+    unit: kubernetes_node
+  - name: resource_id
+    type: string
+  - name: node_capacity_cpu_core_seconds
+    type: double
+    unit: cpu_core_seconds
+  inputs:
+  - name: ReportingStart
+  - name: ReportingEnd
+  query: |
+    SELECT
+      timestamp '{| default .Report.ReportingStart .Report.Inputs.ReportingStart| prestoTimestamp |}' AS period_start,
+      timestamp '{| default .Report.ReportingEnd .Report.Inputs.ReportingEnd | prestoTimestamp |}' AS period_end,
+      node,
+      resource_id,
+      sum(node_capacity_cpu_core_seconds) as node_capacity_cpu_core_seconds
+    FROM {| generationQueryViewName "node-cpu-capacity-raw" |}
+    WHERE "timestamp" >= timestamp '{| default .Report.ReportingStart .Report.Inputs.ReportingStart | prestoTimestamp |}'
+    AND "timestamp" < timestamp '{| default .Report.ReportingEnd .Report.Inputs.ReportingEnd | prestoTimestamp |}'
+    GROUP BY node, resource_id
+    ORDER BY node, resource_id ASC, node_capacity_cpu_core_seconds DESC
+
+---
+
+apiVersion: metering.openshift.io/v1alpha1
+kind: ReportGenerationQuery
+metadata:
+  name: "node-cpu-allocatable-raw"
   labels:
```

Comment:
We have request other places. Are allocatable and request the same use case?

Reply:
No. Allocatable is the amount of a resource a node has available to be requested.